app/src/main/java/com/hedvig/app/service/badge/Seen.kt

```diff
@@ -0,0 +1,13 @@
+package com.hedvig.app.service.badge
+
+@JvmInline
```

Comment:
I think it should recognise value classes, as it probably could also use inline classes when they were named that since Kotlin 1.3. It'd be a long time to still not have any support for it?
As far as the @JvmInline goes, the annotation itself is forced when using the `value` keyword for reasons explained in the [KEEP](https://github.com/Kotlin/KEEP/blob/master/notes/value-classes.md#project-valhalla)